src/__screenshot_tests__/private-skin-components-screenshot-test.tsx

```diff
@@ -0,0 +1,43 @@
+import {openStoryPage} from '../test-utils';
+
+const SKINS = ['Movistar', 'O2', 'Vivo', 'Vivo-new', 'Telefonica', 'Blau'] as const;
```

Comment:
I'd remove Vivo from the list, we are not going to iterate that skin and will eventually be deprecated/replaced by vivo-new

Reply:
removed